aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -1651,19 +1652,9 @@ std::tuple<Tensor&, Tensor&, Tensor&> linalg_lu_factor_ex_out(const Tensor& A,
       singleCheckErrors(info.item<int64_t>(), "torch.linalg.lu_factor_ex");
     }
   }
-
```

Comment:
Should the bit above here go in the meta function too?
```
  if (check_errors) {
    if (A.dim() > 2) {
      batchCheckErrors(info, "torch.linalg.lu_factor_ex");
    } else {
      singleCheckErrors(info.item<int64_t>(), "torch.linalg.lu_factor_ex");
    }
  }
```

Reply:
This checks the error values returned by BLAS / LAPACK. We cannot perform these checks on MetaTensors.

Either that or I did not quite understand what you meant here...

Reply:
oh yup you're right- looks like this check involves actual tensor data, not metadata. nvm